extension/chrome/settings/modules/auth_denied.ts

```diff
@@ -0,0 +1,12 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
+import { Lang } from '../../../js/common/lang.js';
+import { isEnterpriseBuildUsed } from '../../../js/common/helpers.js';
+
+const contactForSupportContainer = $('.ask_support_assistance_container');
+const htmlBody = $('body');
+
+contactForSupportContainer.text(Lang.general.contactIfNeedAssistance(isEnterpriseBuildUsed()));
+htmlBody.css('display', 'block');
```

Comment:
Please write it as a `View` like other pages, `View.run(class AuthDeniedView extends View {` for consistency

Reply:
understand sir... thank you